### ISSUE TITLE
New Ring Bridge Install, No Devices

### ISSUE THREAD

**Issue by DadzBoyz** [dgreif/ring#1462]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe The Bug

My Ring cameras have worked well using this bridge for a long time. They stopped updating, and live view stopped working 5-6 days ago.
* I stopped and restarted the Ring bridge in Hoobs several times.
* I restarted Hoobs several times. No luck.

**Next:**
* Removed the Ring bridge from Hoobs
* Removed the Ring bridge from Apple Home
* Restarted Hoobs
* Reinstalled the Ring bridge
* Linked my Ring account to the bridge (username, password followed by MFA code entered in Ring UI)
* Added the Ring bridge to Apple Home
No doorbell or cameras.

### Expected behavior

Install Ring Bridge in Hoobs
Login to Ring using Configuration settings
Add Ring Bridge to Apple Home
Adding bridge adds doorbell, cameras to Apple Home

### Relevant log output

```shell
8/7/2024, 10:17:34 AMRing Bridge starting
8/7/2024, 10:17:40 AMRing BridgeLoaded plugin 'homebridge-ring'
8/7/2024, 10:17:40 AMRing BridgeLoading 1 platforms...
8/7/2024, 10:17:40 AMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token. fetch is not defined. Trying again in 5 seconds... 8/7/2024, 10:17:40 AMRing BridgeBridge is running on port 51826.
8/7/2024, 10:17:45 AMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token. fetch is not defined. Trying again in 5 seconds... 8/7/2024, 10:17:50 AMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token. fetch is not defined. Trying again in 5 seconds...
```

### Screenshots

![Screenshot 2024-08-07 at 11 08 28 AM](https://github.com/user-attachments/assets/a85bdb28-05e3-4f1f-8a2d-c7518cc53059)
![Screenshot 2024-08-07 at 11 11 10 AM](https://github.com/user-attachments/assets/940c962d-4d59-42e0-bc49-b21bfb658254)
![Screenshot 2024-08-07 at 11 12 00 AM](https://github.com/user-attachments/assets/8672235c-a3f3-4687-8649-6ea8f8706a78)

### Homebridge Ring Config

```shell
{
    "platform": "Ring",
    "refreshToken": 
"Ring token removed as sensitive"
}
```

### OS

Rasbian/Hoobs

### Node.js Version

16.20.2

### NPM Version

?

### Homebridge/HOOBs Version

Homebridge 1.7.0/Hoobs 4.3.1

### Homebridge Ring Plugin Version

13.0.0

### Operating System

Rasbian/Hoobs

**Comment by tsightler** [comment]
Duplicate of https://github.com/dgreif/ring/issues/1459.

TL;DR - HOOBS is outdated and fails to support several newer plugins that depend on newer, and actually supported, NodeJS version.  Upgrading NodeJS is an issue for HOOBS, not an issue with the plugin.  You should contact HOOBS support.

Alternative options are to move away from HOOBS to native Homebridge or continue to use older versions of the plugin that will run on outdated, end-of-life versions of NodeJS.
